package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "asciidoctor-bibtex",
+  "version": "0.8.0",
+  "description": "asciidoctor-bibtex: bibtex integration for AsciiDoc",
+  "main": "dist/main.js",
+  "engines": {
+    "node": ">=8.11",
+    "npm": ">=5.0.0"
+  },
+  "authors": [
+    "ProgramFan (https://github.com/ProgramFan)",
+    "Leon De Andrade (https://github.com/Leon0402)"
+  ],
+  "license": "Open Works",
```

Comment:
Not sure about this licence. 
@mojavelinux do we have guidelines in the Asciidoctor org?

Reply:
I got this from the repo: https://github.com/asciidoctor/asciidoctor-bibtex/blob/master/LICENSE.txt

Reply:
Yes but I'm wondering if we should replace this license to align with other projects in the Asciidoctor organization.

Reply:
@mojavelinux Any news here?

Reply:
The Asciidoctor project doesn't use the Open Works license. It should either be MIT or Apache. I prefer MIT. Open Works is compatible with MIT as far as I can tell (since it is derived from it and extended to non-software) so I don't think there is any issue with updating it.